### PR TITLE
Add circleci-artifacts-redirector-action

### PR DESCRIPTION
**Pull request by larsoner**
@scientific-python/tools-team feel free to merge if it looks okay!

### CODE REVIEW

**Comment by larsoner** [comment]
https://deploy-preview-48--tools-scientific-python-org.netlify.app/actions/

Screenshot 2024-06-05 at 10 35 52 AM

File: content/actions/index.md [hunk 1]
```diff
@@ -48,4 +48,11 @@ body = '''
 Manage teams and team membership for the Scientific Python GitHub organization.
 '''
 
+[[item]]
+type = 'card'
+header = '[circleci-artifacts-redirector-action](https://github.com/scientific-python/circleci-artifacts-redirector-action)'
+body = '''
+Add a GitHub Status link directly to a CircleCI artifact.
```
Comment (j-bowhay) [inline content/actions/index.md:55]: For consistency with the project readme
image

```suggestion
Add a GitHub status link directly to a CircleCI artifact.
```

**Review by j-bowhay — APPROVED** [review]
Looks good, a minor nit that I am also happy to ignore

**Comment by larsoner** [comment]
Makes sense, committed!

**Pull request merged**